misc/python/materialize/feature_benchmark/scenarios/benchmark_main.py

```diff
@@ -1501,6 +1501,93 @@ def benchmark(self) -> MeasurementSource:
         )
 
 
+class MaterializedViewSink(Sink):
+    """Measure the time it takes to emit 1M records to a reuse_topic=true sink. As we have limited
+    means to figure out when the complete output has been emited, we have no option but to re-ingest
+    the data to determine completion.
+    """
+
+    def shared(self) -> Action:
+        return TdAction(
+            self.keyschema()
+            + self.schema()
+            + f"""
+$ kafka-create-topic topic=sink-input partitions=16
+
+$ kafka-ingest format=avro topic=sink-input key-format=avro key-schema=${{keyschema}} schema=${{schema}} repeat={self.n()}
+{{"f1": ${{kafka-ingest.iteration}} }} {{"f2": ${{kafka-ingest.iteration}} }}
+"""
+        )
+
+    def init(self) -> Action:
+        return TdAction(
+            f"""
+>[version<7800]  CREATE CONNECTION IF NOT EXISTS kafka_conn TO KAFKA (BROKER '${{testdrive.kafka-addr}}');
+>[version>=7800] CREATE CONNECTION IF NOT EXISTS kafka_conn TO KAFKA (BROKER '${{testdrive.kafka-addr}}', SECURITY PROTOCOL PLAINTEXT);
+
+> DROP CLUSTER IF EXISTS source_cluster CASCADE
+
+> CREATE CONNECTION IF NOT EXISTS csr_conn
+  FOR CONFLUENT SCHEMA REGISTRY
+  URL '${{testdrive.schema-registry-url}}';
+
+> CREATE CLUSTER source_cluster SIZE '{self._default_size}', REPLICATION FACTOR 1;
+
+> CREATE SOURCE source1
+  IN CLUSTER source_cluster
+  FROM KAFKA CONNECTION kafka_conn (TOPIC 'testdrive-sink-input-${{testdrive.seed}}');
+
+> CREATE TABLE source1_tbl FROM SOURCE source1 (REFERENCE "testdrive-sink-input-${{testdrive.seed}}")
+  FORMAT AVRO USING CONFLUENT SCHEMA REGISTRY CONNECTION csr_conn
+  ENVELOPE UPSERT;
+
+> CREATE MATERIALIZED VIEW mv AS SELECT * FROM source1_tbl;
```

Comment:
I think this gets not created in the default/quickstart cluster. Is that intentional? It might cause the MV to be a bottleneck, since that cluster is probably smaller than `self._default_size`.